.github/workflows/auto-merge-dockers.yml

```diff
@@ -0,0 +1,25 @@
+name: Docker Update Auto Merge
+on:
+  pull_request:
+    types: [opened, edited]
+  schedule:
+    - cron: '0 11 * * *'
+    - cron: '00 14 * * *'
+
+permissions:
+  pull-requests: write
+  contents: write
+
+jobs:
+  auto_approve_and_merge:
+    runs-on: ubuntu-latest
+    if: startsWith(github.head_ref, 'demisto/') && github.repository == 'demisto/content' && contains(github.event.pull_request.labels.*.name, 'docker')
```

Comment:
The filtering of these PRs is done based on a 'docker' label? I think it is too simple let's call it `Docker Image Auto Update`

Reply:
Do you mean to add a new label?

Reply:
Yeah

Reply:
Will create the new label and add it to the flow as well.